src/js/app.js

```diff
@@ -809,6 +811,10 @@ function showProfile(pub) {
     console.log('add friend');
   });
   $('#profile .delete-chat').off().on('click', () => deleteChat(pub));
+  $('input:radio[name=notificationPreference]').off().on('change', (event) => {
+    console.log(`notifications changed to ${event.target.value}`)
```

Comment:
No console.log please :) We could perhaps devise some debug message system where you can set the logging level in console